src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmethod/InputJavadocMethodIgnoreThrows.java

```diff
@@ -0,0 +1,79 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocmethod;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.function.Function;
+
+public class InputJavadocMethodIgnoreThrows {
+
+   /**
+     * Ignore try block, but keep catch and finally blocks.
+     *
+     * @param s String to parse
+     * @return A positive integer
+     */
+    private static int parsePositiveInt(String s) {
+        try {
+            int value = Integer.parseInt(s);
+            if (value <= 0) {
+                throw new NumberFormatException(value + " is negative/zero"); // ok, try
+            }
+            return value;
+        } catch (NumberFormatException ex) {
+            throw new IllegalArgumentException("Invalid number", ex); // violation, catch
+        } finally {
+            throw new IllegalStateException("Should never reach here"); // violation, finally
+        }
+    }
+
+    /**
+     * Ignore everything inside lambda.
+     *
+     * @param maxLength Max length
+     * @return A function to truncate string
+     */
+    private static Function<String, String> getTruncateFunction(int maxLength) {
+        return s -> {
+            if (s == null) {
+                throw new IllegalArgumentException("Cannot truncate null"); // ok, inside lambda
+            }
+            return s.length() > maxLength ? s.substring(0, maxLength) : s;
+        };
+    }
+
+    /**
+     * Try-with-resources should also be ignored if there is a catch block.
+     *
+     * @param input file to read
+     */
+    private static void ignoreTryWithResources(String input) {
+        try (BufferedReader in = new BufferedReader(new FileReader(input))) {
+            String s = in.readLine();
+            System.out.println(s);
+            if (s.length() == 0) {
+                // false negative, unable to tell what was caught
+                throw new IllegalArgumentException("empty input");
+            }
+            else {
+                throw new IOException(); // ok, exception was caught
+            }
+        } catch (IOException e) {
+            System.out.println("Error reading file");
+        }
+    }
+
+    /**
+     * However, do not ignore try block without catch.
+     */
+    private static void keepTryWithoutCatch() {
+        try (Scanner sc = new Scanner(System.in)) {
+            if (sc.nextInt() <= 0) {
+                // violation, not caught and no @param
+                throw new IllegalArgumentException("Please enter a positive integer");
```

Comment:
please put "// violation" comment on the same line where violation is placed. One day we will make test to validate this.
Extra explanation you can put in other lines, but it is better to shorten code `new Exception(""); // violation ....`

Reply:
Fixed